sirepo/package_data/static/js/srw.js

```diff
@@ -2596,6 +2596,16 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 return true;
             }
 
+            function logFileRequest(suffix) {
+                return  requestSender.formatUrl('downloadDataFile', {
+                    '<simulation_id>': appState.models.simulation.simulationId,
+                    '<simulation_type>': SIREPO.APP_SCHEMA.simulationType,
+                    '<model>': $scope.simState.model,
+                    '<frame>': -1,
```

Comment:
This really should be a constant....

Reply:
constant declared where? if outside of `logFileRequest()` but in either `progressLogUrl()` or `logFileUrl()`, then it needs to be in both. If outside of all and declared at the top of the directive, then that wont work. If inside `logFileRequest()` then I am not sure I understand the point.

Reply:
-1 is something and should be documented as such:

```sh
$ grep -i -- frame.*.-1 *.js
opal.js:            '<frame>': -1,
radia.js:                    '<frame>': -1,
rs4pi.js:                        $scope.advanceFrame(-1);
sirepo-plotting.js:        scope.prevFrameIndex = -1;
sirepo-plotting.js:            scope.prevFrameIndex = -1;
sirepo-plotting.js:                    scope.prevFrameIndex = -1;
sirepo-plotting.js:                    scope.prevFrameIndex = -1;
sirepo-plotting.js:              <button type="button" class="btn btn-default" data-ng-disabled="isFirstFrame()" data-ng-click="advanceFrame(-1, true)"><span class="glyphicon gl\
yphicon-step-backward"></span></button>
sirepo-plotting.js:                $scope.prevFrameIndex = -1;
srw.js:                '<frame>': -1,
srw.js:                    '<frame>': -1,
zgoubi.js:                    '<frame>': appState.isAnimationModelName(modelKey) ? 1 : -1,
```

Reply:
oh. Thansk for clarifying 👍  I thought you meant the whole object. I'll make it a constant.